tools/tempfiles.py

```diff
@@ -11,9 +11,17 @@
 import sys
 
 
+def try_delete(pathnames):
+  if isinstance(pathnames, list):
+    for pathname in pathnames:
+      try_delete_one(pathname)
+  else:
+    try_delete_one(pathnames)
```

Comment:
I'd rather not add helpers like this that are only used in test code.   Also I'm not a fan of this type of polymorphism in python.    How about we have a new helper to in `test_other.py` called `delete_files`?